cupy/cuda/cub.pyx

```diff
@@ -385,6 +421,23 @@ def cub_reduction(arr, op, axis=None, dtype=None, out=None, keepdims=False):
     return None
 
 
+def cub_scan(arr, op):
+    """Perform an (in-place) prefix scan using CUB.
+
+    If the specified scan is not possible, None is returned.
+    """
+    if op < CUPY_CUB_CUMSUM or op > CUPY_CUB_CUMPROD:
```

Comment:
`if op not in (CUPY_CUB_CUMSUM, CUPY_CUB_CUMPROD):`

Reply:
I will fix it later in my pr

Reply:
Oh, sorry for overlooking this @emcastillo 😓